src/main/java/com/iexec/resultproxy/proxy/ProxyController.java

```diff
@@ -73,6 +73,18 @@ public ResponseEntity<String> login(@RequestParam(name = "chainId") Integer chai
         return ResponseEntity.ok(jwtString);
     }
 
+    /**
+     * Push result on IPFS through iExec Result Proxy.
+     *
+     * @param token JWT authorization
+     * @param model Result payload containing the bytes to push on IPFS
+     * @return A response entity indicating the status and details of the operation
+     * <ul>
+     * <li>HTTP 200 (OK) - If the result file was pushed on IPFS. The multihash will be included in the response body.
+     * <li>HTTP 400 (BAD REQUEST) - If the operation was authorized but the file could not be pushed.
+     * <li>HTTP 401 (UNAUTHORIZED) - If the operation is forbidden
```

Comment:
We could use the same sentence as for the '/results/{chainTaskId}' API (If the client is not authorized to query the information) both for consistency and to avoid using the word forbidden, which is associated with a 403.

Reply:
javadoc updated